Code4SocialGood/c4sg-web
# [issue-126] Create Organization Page - Page Forward

To test,
1) Create an org for admin user
2) Check after create it redirects to view org details page.

Note : There are few issues with this
1) after clicking create button it takes some time to create an org. I think it has some backend issue because  at the same time eclipse gets hang for a while.
2) View org detail page is showing some data like city & zip which are not available in getOrganization() response that's why it shows undefined.
